app_spec/test/default.nix

```diff
@@ -26,7 +26,24 @@ let
   hc-conductor-install
   (cd app_spec && APP_SPEC_NETWORK_TYPE="sim1h" ./build_and_test.sh);
   '';
+
+  name-sim2h = "hc-app-spec-test-sim2h";
+
+  script-sim2h = pkgs.writeShellScriptBin name-sim2h ''
+  set -euo pipefail
+  hc-cli-install
+  hc-conductor-install
+  (cd app_spec && APP_SPEC_NETWORK_TYPE="sim2h" ./build_and_test.sh);
+  '';
+
+  name-sim2h-server = "hc-sim2h-server";
+
+  script-sim2h-server = pkgs.writeShellScriptBin name-sim2h-server ''
+  set -euo pipefail
+  cargo install sim2h_server -f --git https://github.com/holochain/sim2h
+  RUST_LOG=debug sim2h_server
```

Comment:
It would be more flexible to allow the port to be passed in optionally from the outside for sim2h, I guess it's likely been categorized as a 'thing for later'

Reply:
actually this is worth adding over on the sim2h repo.  But I will make it so if you don't put anything on the command line it will default to port 9000 so we don't have to touch this code.

Reply:
done:  if you want now you can pass in the port with --port=xxx

Reply:
Nice, yeah, that's what I was thinking... Do It on the sim2h side.